petab/parameter_mapping.py

```diff
@@ -134,19 +179,24 @@ def get_parameter_mapping_for_condition(
     """
     _perform_mapping_checks(cur_measurement_df)
 
-    par_sim_ids = sbml.get_model_parameters(sbml_model)
+    if simulation_parameter_ids is not None and sbml_model is None:
+        pass
+    elif simulation_parameter_ids is None and sbml_model is not None:
+        simulation_parameter_ids = sbml.get_model_parameters(sbml_model)
+    else:
+        raise ValueError("Must provide exactly one of `sbml_model` and "
+                         "`simulation_parameter_ids`.")
```

Comment:
can be shortened: if pid is None and model is not None: ...; elif (pid is None) == (model is None): Error

Reply:
You are right, but found the current way easier to read.